rfd/0189-windows-access-guide.md

```diff
@@ -0,0 +1,105 @@
+---
+authors: Dave Sudia (david.sudia@goteleport.com)
+state: draft
+---
+
+# RFD 0189 - Windows Desktop Access guide
+
+## Required Approvers
+
+* Engineering: @r0mant
+* Product: @xinding33
+
+## Overview
+
+### Problem
+
+* Currently only 15% of users convert from going to the docs to starting a session
+* We need better insight into drop-off points: we only know they clicked out to the docs then started a session and don’t know where they’re struggling
+
+### Other data/context
+
+Historically we have more local user sessions than AD sessions (last 180 days, 36.3k local vs 30.1k AD). This may be skewed by us not getting metrics from major self-hosted customers who are likely using AD.
```

Comment:
```suggestion
Historically Teleport Cloud has more local user sessions than AD sessions (last 180 days, 36.3k local vs 30.1k AD). This may be skewed by us not getting metrics from major self-hosted customers who are likely using AD.
```